app/site/metrics.js

```diff
@@ -22,144 +22,192 @@ export const communityMetrics = {
   rideshare_pickups_covid: {
     name: "Rideshare Pickups Since March 2020",
     units: "trips",
+    dataset: "Rideshare",
+    description: "",
     format: Formatter.numberInMillions,
     fullFormat: Formatter.numberWithCommas,
   },
   rideshare_pooled_trip_rate_2018: {
     name: "2018 Rideshare Pooled Trip Rate",
     units: "of trips",
+    dataset: "Rideshare",
+    description: "",
     format: Formatter.percentWithNoDecimal,
     fullFormat: Formatter.percentWithOneDecimal,
   },
   rideshare_pooled_trip_rate_2019: {
     name: "2019 Rideshare Pooled Trip Rate",
     units: "of trips",
+    dataset: "Rideshare",
+    description: "",
     format: Formatter.percentWithNoDecimal,
     fullFormat: Formatter.percentWithOneDecimal,
   },
   rideshare_pool_request_rate_2018: {
     name: "2018 Rideshare Pool Request Rate",
     units: "of trips",
+    dataset: "Rideshare",
+    description: "",
     format: Formatter.percentWithNoDecimal,
     fullFormat: Formatter.percentWithOneDecimal,
   },
   rideshare_pool_request_rate_2019: {
     name: "2019 Rideshare Pool Request Rate",
     units: "of trips",
+    dataset: "Rideshare",
+    description: "",
     format: Formatter.percentWithNoDecimal,
     fullFormat: Formatter.percentWithOneDecimal,
   },
   total_population_2010: {
     name: "2010 Total Population",
     units: "people",
+    dataset: "Population",
+    description: "Average population over the time period that ends on 2010.",
     format: Formatter.numberInThousands,
     fullFormat: Formatter.numberWithCommas,
   },
   total_population_2019: {
     name: "2019 Total Population",
     units: "people",
+    dataset: "Population",
+    description: "Average population over the time period that ends on 2010.",
     format: Formatter.numberInThousands,
     fullFormat: Formatter.numberWithCommas,
   },
   median_income_2010: {
     name: "2010 Median Income",
     units: "dollars",
+    dataset: "Income",
+    description: "Income in the past 12 months for 2010, in inflation-adjusted 2017 dollars.",
```

Comment:
Thanks for adding these! I can fill in the others. We'll also want to add `source`, but that can be done in another PR.